pkg/server/server.go

```diff
@@ -46,7 +48,27 @@ func (s *ServerConfig) StartAndListen() {
 		TLSNextProto:      make(map[string]func(*http.Server, *tls.Conn, http.Handler)),
 	}
 
-	klog.Info("Server listening on: ", srv.Addr)
-	klog.Fatal(srv.ListenAndServeTLS("./sslcert/tls.crt", "./sslcert/tls.key"),
-		" Use ./setup.sh to generate certificates for local development.")
+	// Start the server
+	go func() {
+		klog.Info("Listening on: ", srv.Addr)
+		// ErrServerClosed is returned on graceful close.
+		if err := srv.ListenAndServeTLS("./sslcert/tls.crt", "./sslcert/tls.key"); err != http.ErrServerClosed {
+			if config.Cfg.DevelopmentMode {
+				log.Fatal(err, ". If missing certificates in development mode, use ./setup.sh to generate.")
```

Comment:
Why do we switch to log, instead of klog, here?

Reply:
it was a typo.  Changed to klog.